src/components/lib/VDropdown/VDropdown.vue

```diff
@@ -11,7 +11,7 @@ const props = defineProps({
     default: '',
   },
   modelValue: {
-    type: [String, Object],
+    type: [String, Object, Boolean],
```

Comment:
why `Object` type?

Reply:
We also have cases where we send `{label: 'Dark Mode', value: 'dark'}`. This is tend to improve when we start creating component library